pkg/networkmanager/dialogs-common.jsx

```diff
@@ -212,6 +230,8 @@
             dlg = <TeamDialog {...properties} />;
         else if (type == 'bridge')
             dlg = <BridgeDialog {...properties} />;
+        else if (type == 'wireguard')
+            dlg = <WireGuardDialog {...properties} />;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19024-20230711-114115-3bc11384-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)